src/EditorFeatures/CSharpTest/EmbeddedLanguages/RegularExpressions/CSharpRegexParserTests.cs

```diff
@@ -228,12 +228,35 @@ private static XAttribute GetTextAttribute(SourceText text, TextSpan span)
 
         private XElement NodeToElement(RegexNode node)
         {
-            var element = new XElement(node.Kind.ToString());
-            foreach (var child in node)
+            if (node is RegexAlternationNode alternationNode)
+            {
+                return AlternationToElement(alternationNode, alternationNode.SequenceList.NodesAndTokens.Length);
+            }
```

Comment:
this change in the test harness was so that we could keep all our current baselines.  without this, 90% of all tests needed a baseline change.